infra/fendermint/scripts/fendermint.toml

```diff
@@ -34,7 +34,8 @@ docker run \
   --env FM_SNAPSHOTS_DIR=/data/${NODE_NAME}/fendermint/snapshots \
   --env FM_CHAIN_NAME=${NETWORK_NAME} \
   --env TENDERMINT_RPC_URL=http://${CMT_CONTAINER_NAME}:26657 \
-  --env IROH_RPC_ADDR=${IROH_CONTAINER_NAME}:4919 \
+  --env IROH_PATH=/data/${NODE_NAME}/iroh-fendermint \
+  --env IROH_RPC_ADDR=0.0.0.0:4919 \
```

Comment:
Is it correct that `IROH_RPC_ADDR=0.0.0.0`? Shouldn't it be like before `IROH_RPC_ADDR=${IROH_CONTAINER_NAME}:4919`? Maybe I'm missing something.

Reply:
this is the bind address of the rpc, which is what gets passed to the objects api